scripts/should-rebuild-on-vercel.sh

```diff
@@ -114,31 +112,32 @@ get_all_changed_files () {
 }
 
 #
-# We're only interested in checking the examples and packages folders, ignore
+# We're only interested in checking some folders, ignore
 # any other changed files.
 #
 get_interesting_changed_files () {
     get_all_changed_files \
       | grep -Ee '^examples/' \
+      | grep -Ee '^starter-kits/' \
```

Comment:
This won't work in a grep pipe. It's like doing

```getAllFiles
  .filter(f => f.startsWith('examples/'))
  .filter(f => f.startsWith('starter-kits/'))
```

That will always give 0 results.

You should say the equivalent of:

```getAllFiles
  .filter(f => f.startsWith('examples/') || f.startsWith('starter-kits/'))
```

That can be done using:

```suggestion
      | grep -Ee '^(examples|starter-kits)/' \
```

Reply:
Thanks for the fix and the explanation!